gobblin-modules/gobblin-couchbase/scripts/install_test_deps.sh

```diff
@@ -10,5 +10,9 @@ if [ -d mock-couchbase/.git ];
     git -C mock-couchbase checkout b929a9f99a31a233c43bd94b6a696ad966206e02
   fi
 pushd mock-couchbase
-mvn package -Dmaven.test.skip=true
-popd
+if mvn package -Dmaven.test.skip=true; then
+  popd
```

Comment:
Oh okay, thanks! When I ran 'gradlew build', the end directory seemed to be undisturbed.